packages/config/src/schema.js

```diff
@@ -0,0 +1,189 @@
+'use strict';
+
+const castArray = require('lodash/castArray');
+const filter = require('lodash/filter');
+const find = require('lodash/find');
+const first = require('lodash/first');
+const flatMap = require('lodash/flatMap');
+const get = require('lodash/get');
+const isString = require('lodash/isString');
+const map = require('lodash/map');
+const reduce = require('lodash/reduce');
+const sortBy = require('lodash/sortBy');
+const union = require('lodash/union');
+const uniq = require('lodash/uniq');
+
+module.exports = (schemas, defaultConfiguration) => {
+  return {
+    getSchemaId,
+    getSchema,
+    getLevel: getActivityConfig,
```

Comment:
Please introduce getLevel deprecation and use getActivityConfig (Can be a separate PR)